.github/workflows/e2e_cron.yml

```diff
@@ -45,8 +45,8 @@ jobs:
 
       - name: Run Playwright tests - ${{ inputs.environment != null && inputs.environment || 'stage' }}
         if: github.actor != 'dependabot[bot]'
-        run: npm run e2e
-        timeout-minutes: 20
+        run: npm run e2e -- --fully-parallel=false --workers=1
```

Comment:
Is this the same issue for the full smoke test for PR?

Reply:
No, not the exact same issue. With the smoke test suite, it was more around setting the allowed max test failures.